### PR TITLE
Burger menu

### PR DESCRIPTION
**Pull request by DenysShurubura**

### CODE REVIEW

**Review by OlhaKoziuk — APPROVED** [review]
Unfortunatly I cant see how it works, the code looks nice

File: src/modules/shared/components/Header/Header.tsx [hunk 1]
```diff
@@ -80,9 +87,64 @@ export const Header: React.FC = () => {
...
+            <ul className={styles.header__burger_menu}>
+              <li>
+                <NavLink
+                  to="/"
+                  className={getLinkClass}
+                  onClick={() => setIsBurgerActive(false)}
+                >
+                  home
+                </NavLink>
+              </li>
```
Comment (dsfreedom) [inline src/modules/shared/components/Header/Header.tsx:115]: Maybe adding spaces would make it a bit more better looking

**Review by dsfreedom — APPROVED** [review]
Same story as with Olha, the code looks good

**Comment by alexandra-protyanova** [comment]
UI changes can't be properly reviewd without screenshots or videos, provide them please

**Pull request merged**